tests/api.c

```diff
@@ -27714,6 +27714,7 @@ static void test_wolfSSL_PKCS7_certs(void)
         if (i == 0) {
             PKCS7_free(p7);
             AssertNotNull(d2i_PKCS7(&p7, &p, buflen));
+            AssertIntEQ(wc_PKCS7_VerifySignedData(p7, ((WOLFSSL_PKCS7*)p7)->data, ((WOLFSSL_PKCS7*)p7)->len), 0);
```

Comment:
Please leave this as it was and remove the `wc_PKCS7_VerifySignedData` call after `d2i_PKCS7`. This will break support for projects that use `d2i_PKCS7` and expect it to properly initialize the internal structure as well. The tests in `api.c` should make the same calls as OpenSSL users.

Reply:
Fixed.